src/System.IO.Abstractions.TestingHelpers/MockDirectoryInfo.cs

```diff
@@ -26,6 +27,15 @@ public MockDirectoryInfo(IMockFileDataAccessor mockFileDataAccessor, string dire
         {
             this.mockFileDataAccessor = mockFileDataAccessor ?? throw new ArgumentNullException(nameof(mockFileDataAccessor));
 
+            if (directoryPath== null)
```

Comment:
Style, spaces around ` == `
```suggestion
            if (directoryPath == null)
```

Reply:
Fixed in latest commit